src/DonationForms/V2/ServiceProvider.php

```diff
@@ -47,14 +47,7 @@ public function boot()
 
         // Onboarding
         Hooks::addAction('submitpost_box', DonationFormsAdminPage::class, 'renderMigrationGuideBox');
-
-        if (DonationFormsAdminPage::isShowingEditV2FormPage()) {
-            Hooks::addAction('admin_enqueue_scripts', DonationFormsAdminPage::class, 'loadEditFormScripts');
-        }
-
-        if (DonationFormsAdminPage::isShowingAddV2FormPage()) {
-            Hooks::addAction('admin_enqueue_scripts', DonationFormsAdminPage::class, 'loadAddFormScripts');
-        }
```

Comment:
Why are these being removed?

Reply:
This is related to migration onboarding stuff. The problem was that the logic to check if the current page is a donation form page was not functioning correctly and migration onboarding JS was loaded on regular pages causing JS errors in the console. The error was annoying so I had to fix it :smile:

Reply:
Sounds good, thanks for the clarification.  Let's just make sure to add this to the QA testing instructions.